routes/tweet.js

```diff
@@ -1,32 +1,270 @@
 const router = require("express").Router();
 let Tweet = require("../models/tweet.model");
+let User = require("../models/user.model");
+var ObjectID = require('mongodb').ObjectID;
 
-router.route("/").get((req, res) => {
-  Tweet.find()
+router.route("/getUserTweets").get((req, res) => {
```

Comment:
I dont think we should call this getUserTweets because as far as I can tell it returns all tweets sorted by created_at descending. The name kinda implies it is a route to get all tweets made by a particular user.

Maybe we can call it /getAllTweets